lib/betamocks/middleware.rb

```diff
@@ -16,12 +16,13 @@ def call(env)
         @response_cache = Betamocks::ResponseCache.new(env: env, config: @endpoint_config)
         response = @response_cache.load_response
         return response if response
+        return @response_cache.load_response('default.yml') if Betamocks.configuration.mode == Betamocks::Configuration::PLAYBACK
       end
       super
     end
 
     def on_complete(env)
-      return unless Betamocks.configuration.enabled
+      return unless Betamocks.configuration.enabled && Betamocks.configuration.mode == Betamocks::Configuration::RECORDING
```

Comment:
same here `unless Betamocks.configuration.enabled && Betamocks.configuration.recording?` might even be able to move the `Betamocks.configuration.enabled` part into the `Betamocks.configuration.recording?` method because it has to be enabled to record.